install/update_2.6_2.7.php

```diff
@@ -32,6 +32,18 @@
 function plugin_formcreator_update_2_7(Migration $migration) {
    global $DB;
 
+   // Upgrade plugin configuration table
+   $table = 'glpi_plugin_formcreator_entityconfigs';
+   $migration->displayMessage("Upgrade $table");
+   $migration->addField($table, 'external_links_prefix', 'string', ['after' => 'id']);
+   $migration->addField($table, 'external_links_icon', 'string', ['after' => 'id']);
+   $migration->addField($table, 'external_links_title', 'string', ['after' => 'id']);
+   $migration->addField($table, 'tickets_summary', 'integer', ['after' => 'id', 'value' => '1']);
+   $migration->addField($table, 'user_preferences', 'integer', ['after' => 'id', 'value' => '1']);
+   $migration->addField($table, 'avatar', 'integer', ['after' => 'id', 'value' => '1']);
+   $migration->addField($table, 'user_name', 'integer', ['after' => 'id', 'value' => '0']);
+   $migration->addField($table, 'profile_selector', 'integer', ['after' => 'id', 'value' => '1']);
+
```

Comment:
In all those new lines you add fields after the column ID. I think the order of the columns after an upgrade wwill be the reverse of the order of columns after a fresh install.

The following should ensure that the columns are corectly ordered. It seems that unit tests pass at the moment, maybe because I did not yet added upgrade tests (i thought I did, i need to double check). 

```suggestion
   // Upgrade plugin configuration table
   $table = 'glpi_plugin_formcreator_entityconfigs';
   $migration->displayMessage("Upgrade $table");
   $migration->addField($table, 'external_links_prefix', 'string', ['after' => 'replace_helpdesk']);
   $migration->addField($table, 'external_links_icon', 'string', ['after' => 'external_links_prefix']);
   $migration->addField($table, 'external_links_title', 'string', ['after' => 'external_links_icon']);
   $migration->addField($table, 'tickets_summary', 'integer', ['after' => 'external_links_title', 'value' => '1']);
   $migration->addField($table, 'user_preferences', 'integer', ['after' => 'tickets_summary', 'value' => '1']);
   $migration->addField($table, 'avatar', 'integer', ['after' => 'user_preferences', 'value' => '1']);
   $migration->addField($table, 'user_name', 'integer', ['after' => 'avatar', 'value' => '0']);
   $migration->addField($table, 'profile_selector', 'integer', ['after' => 'user_name', 'value' => '1']);
```